Client/Frontend/Menu/Items/MenuItemView.swift

```diff
@@ -0,0 +1,106 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import UIKit
+
+/**
+ * creates a view that consists of an image and a title.
+ * the image view is displayed at the center top of the
+ **/
+public class MenuItemView: UIControl {
+
+    var padding: CGFloat = 5.0
+
+    private(set) public var imageView: UIImageView
+    private(set) public var titleLabel: UILabel
+
+    private var previousLocation: CGPoint?
+    public init() {
+        imageView = UIImageView()
+        titleLabel = UILabel()
+
+        super.init(frame: CGRectZero)
+
+        titleLabel.lineBreakMode = NSLineBreakMode.ByWordWrapping
+        titleLabel.numberOfLines = 2
```

Comment:
I'm a bit worried constraining this to two lines. It might be better to have it set to 0 lines and have it flow instead. Not sure how this will work with long text locales.